### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Byrdman32**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- Array     > `a`  > Example: `int aExampleArray[4]`
+- String    > `sz` > Example: `string szExampleString`
+- Pointer   > `p`  > Example: `string* pExamplePointer`
+- Enum      > `e`  > Example: `ExampleEnum eExampleUseOfEnum`
+- Time      > `tm`  > Example: `time_t tmExampleUseOfTime`
+- Tuple      > `tp`  > Example: `tuple tpExampleUseOfTuple`
+- Thread      > `th`  > Example: `jthread thExampleUseOfThread`
+
+#### External Types
+- OpenCV > `cv`  > Example: `bool bExampleBoolean`
```
Comment (ClayJay3) [inline CONTRIBUTING.md]: Is the example for this one correct? It's the same as the normal boolean.

**Review by ClayJay3 — COMMENTED** [review]
See comment on CONTRIBUTING.md
I also added time, tuple, thread notations.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Byrdman32) [reply CONTRIBUTING.md]: Copy and paste error

**Review by ClayJay3 — APPROVED** [review]
Approved.

**Pull request merged**